unittest/CMakeLists.txt

```diff
@@ -44,6 +44,13 @@ SET(${PROJECT_NAME}_CPP_TESTS
   test_solvers
 )
 
+# test_diff_actions fails with Clang, cf. https://github.com/loco-3d/crocoddyl/issues/964
+# It's difficult to identify the source - it appears to be a double free from inside boost's unittest framework.
+# Hence, we will remove it for now, it's still tested with g++:
+IF(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
```

Comment:
If this works locally and not in the buildserver, wouldn't be better to have a flag for the buildserver (which is disable by default)?

This allows developers to run this test in their PCs.

Reply:
I can reproduce it when running from a Docker in the same way the buildserver is, but not on my local installation. Locally I am using a slightly different version of Pinocchio and other dependencies, so it might be that. If you confirm that commenting those lines it works on your setup, happy to make it conditional for the buildserver - otherwise this is the easiest and will be tested on the buildserver nonetheless with g++